src/main/java/com/boot/user/controller/UserController.java

```diff
@@ -105,6 +104,13 @@ public ResponseEntity<UserDTO> deleteUserByByEmail(@Email(message = "Invalid ema
         return new ResponseEntity<>(HttpStatus.OK);
     }
 
+    @Operation(summary = "Save customer message", tags = {"customerMessage"})
+    @PostMapping(value = "/customerMessage")
+    public ResponseEntity<String> sendEmailFromUser(@RequestBody CustomerMessage customerMessage) {
```

Comment:
If you want to use the model for sending data add validation for it: email validity, 
name, phone number, comment, name - length
Also, in order to enable validation add @Valid annotation